week-03/README.md

```diff
@@ -16,69 +16,95 @@ The instructor will then review the pull request and make comments should furthe
 
 
 [Course Textbook Chapter 3, Problem 1]
+- *Question* (sbagdi-stat6250): Which of the following is true for SAS statements?
```

Comment:
Invalid question for a Weekly Quiz; you're not creating a quiz for someone else. The instructions specify that you should be asking questions about SAS programming that **you** actually have, as prompted by concepts in textbook problems and recipes.